### DISCUSSION TITLE
Related Table column shows id, how do I pick a different column to show from the related table

### DISCUSSION THREAD

**Discussion by bjg-dev** [Budibase/budibase#2941] [Help]
The Details:
I am making a little app to manage tasks. Each task can have a category. The category table is a one to many related table with regard to the task table. When I add a record to the task table all I get is the id to select from and have no idea which category I am selecting. The same thing goes for viewing the task table, I do not know which category is selected, but just the id shows up. Is there a way to display a different column than the id column in these places? 

Thanks in advance for the help!

**Comment by mjashanks** [comment]
Hi @bjg-dev - could you provide a little more info? I can get this to work ok.

Are you using the Budibase internal table, or SQL? 
Could you provide some screenshots?
When you say you are "viewing the task table" - from where are you viewing it? In the `Data` section, for via the app that you have built?
